website/content/docs/bloop-basics.md

```diff
@@ -51,9 +54,16 @@ get peak performance at all times.
 By supporting popular build tools out of the box and different scenarios,
 `bloop` enforces by design best practices to be productive.
 
-### What bloop is not
+#### Do less work
+
+Another way Bloop decreases gives a snappy developer workflow is by doing
+less work. Bloop only focuses on compiling, testing and running your code.
+Everything else is left to external tools so that only the blocking tasks are
+on your way when editing code.
+
+### Not a build tool
 
-Bloop is **not** a new build tool, but rather a foundation block for Scala
+Bloop is not a new build tool, but rather a foundation block for Scala
```

Comment:
s/new//

Reply:
>  but rather a foundation block for Scala.
> ...
>  It can be used by Scala developers to be more productive and by tooling authors to write tools in a fast and reliable way.

This paragraph does not explain why bloop is "not a build tool". I would write this section from scratch explaining in detail why bloop is not a build tool, I suspect many will be interested to hear your arguments.

The "foundation" point belongs to another paragraph about "Foundation for Scala tooling". It's a bold claim that's currently not backed by any material arguments, I recommend you drop it altogether because I'm skeptical of the claim.

Reply:
> This paragraph does not explain why bloop is "not a build tool". I would write this section from scratch explaining in detail why bloop is not a build tool, I suspect many will be interested to hear your arguments.
  
I'm aware, this is why I said I'll send a PR to fix it soon.